homeassistant/components/google_assistant/strings.json

```diff
@@ -4,6 +4,28 @@
       "sync_devices": {
         "name": "Synchronize devices"
       }
+    },
+    "fan": {
+      "speed": {
+        "name": "Speed",
+        "state": {
```

Comment:
This is somewhat weird. It's a state that generally does not exist. But it is a way to add translation to virtual data. If you have some other suggestion for this, i would be obliged.